docs/quick_start.md

```diff
@@ -11,7 +11,13 @@ It assumes that you have already [installed `jstz`](installation.md) and have a
 With `jstz` you can deploy so called _smart functions_ which are operating similarly to cloud functions, while running on Tezos L2 and
 providing additional security and blockchain-specific functionality typical for smart contracts.
 
-## 1) Your First Smart Function
+## 1. Your First Smart Function
+
```

Comment:
I disagree, it should not be part of the installer. If we had the time, it would be part of the jstz CLI to have a `template` command -- but for now, manually cloning doesn't hurt